### PR TITLE
Prepare for initial release 0.1a.x

### PR DESCRIPTION
**Pull request by lzchen**
Adding CHANGELOG.md file and merging changes from master into release branch.

I'll be doing a release today.

### CODE REVIEW

File: azure_monitor/setup.cfg [hunk 1]
```diff
@@ -27,8 +27,8 @@ package_dir=
     =src
 packages=find_namespace:
 install_requires =
-    opentelemetry-api >= 0.1a.0
-    opentelemetry-sdk >= 0.1a.0
+    opentelemetry-api >= 0.2a.0
```
Comment (hectorhdzg) [inline azure_monitor/setup.cfg:30]: Latest is 0.3.dev0

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Code owners file.
```
Comment (hectorhdzg) [inline .github/CODEOWNERS:1]: I have déjà vu about this PR :)

Comment (lzchen) [reply .github/CODEOWNERS:1]: Yeah, we have to add it separately to each branch. Just initial overhead.

File: azure_monitor/setup.cfg (hunk 1 above)
Comment (lzchen) [reply azure_monitor/setup.cfg:30]: The package dependencies depend on `opentelemetry-api` and `opentelemetry-sdk` versions that are RELEASED, not on dev.
https://pypi.org/project/opentelemetry-sdk/
https://pypi.org/project/opentelemetry-api/

**Review by hectorhdzg — APPROVED** [review]

**Pull request merged**